Abstract (#21)
greenelab/knowledge-graph-review: content/01.abstract.md

## Abstract {.page_break_before}

> Knowledge graphs are a practical resource for many biomedical applications.

Reviewer: ```suggestion
Knowledge graphs can support many biomedical applications.
```

These graphs represent biomedical concepts and relationships in the form of nodes and edges.

> These graphs are constructed from pre-existing databases that are populated by experts via manual curation and automatic systems.

Reviewer: ```suggestion
In this review we discuss how these graphs are constructed and applied with a particular focus on the ways that machine learning approaches are changing these processes.
In many examples in the literature, biomedical knowledge graphs have been constructed from pre-existing databases that are populated by experts via manual curation, but we are now also seeing a more robust use of automatic systems.
```

> Following construction, these graphs are represented in a low dimensional space via representation learning.

Reviewer: ```suggestion
A number of techniques are used to represent knowledge graphs, but often machine learning methods are used to learn a low-dimensional representation that can support many different applications.
```

> This space preserves a knowledge graph’s local and/or global structure and can support efforts to apply machine learning methods to make predictions within the multi-omics, pharmaceutical, and clinical domains.

Reviewer: ```suggestion
This representation is designed to preserve a knowledge graph’s local and/or global structure.
Additional machine learning methods can be applied to this representation to make predictions within genomic, pharmaceutical, and clinical domains.
```

> In this review we highlight the practicality of knowledge graphs within the biomedical realm.

Reviewer: ```suggestion
We frame our discussion first around construction before discussing unifying techniques and unifying applications separately.
Advances in machine learning for biomedicine are creating new opportunities across many domains, and we note potential avenues for future work with knowledge graphs that appear particularly fruitful.
```

> We first discuss different ways to construct a knowledge graph and provide future adive to improve these processes.

Reviewer: ```suggestion
```

> Next, we discuss the pros and cons of representation learning techniques that project knowledge graphs onto a low dimensional space.

Reviewer: ```suggestion
```

> Finally, we talk about applications that use knowledge graphs to perform biomedical tasks and propose future opportunities for these applications.

Reviewer: ```suggestion
```
